protobuf/protobuf.go

```diff
@@ -0,0 +1,150 @@
+package protobuf
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// Package represents an unique .proto file with its own package definition.
+type Package struct {
+	Name     string
+	Path     string
+	Imports  []string
```

Comment:
Actually, it wasn't because of the external package, it's just because right now having the reference of the packages is not necessary for any step and it was just to keep track of the files you had to import when generating the actual `.proto` file. But yeah, they can be references instead, although it is not used right now.